ground/src/main/java/com/google/android/ground/ui/home/HomeScreenViewModel.kt

```diff
@@ -88,7 +91,13 @@ internal constructor(
   }
 
   fun showOfflineAreas() {
-    navigator.navigate(HomeScreenFragmentDirections.showOfflineAreas())
+    viewModelScope.launch {
+      val count = offlineAreaRepository.offlineAreas().first().count()
```

Comment:
Would it be more idiomatic to use `isEmpty()` and invert the if statement below instead of `first().count() > 0`?

Reply:
Done